devito/symbolics/manipulation.py

```diff
@@ -160,11 +163,15 @@ def pow_to_mul(expr):
         return expr
     elif expr.is_Pow:
         base, exp = expr.as_base_exp()
-        if exp <= 0 or not exp.is_integer:
-            # Cannot handle powers containing non-integer non-positive exponents
+        if exp > 10 or exp < -10 or int(exp) != exp or exp == 0:
+            # Large and non-integer powers remain untouched
             return expr
-        else:
+        elif exp > 0:
             return sympy.Mul(*[base]*exp, evaluate=False)
+        else:
+            # sympy represents 1/x as Pow(x,-1)
+            posexpr = sympy.Mul(*[base]*(-exp), evaluate=False)
+            return sympy.Pow(posexpr, -1, evaluate=False)
```

Comment:
Am I beeing stupid or is it the same as `expr` input

Reply:
It should convert negative integer powers into the combo of multiplication and reciprocal

Reply:
To be fair, it does nothing for exp==-1, so we could catch that in the untouched clause higher up.